testdata/client/connecth1unary.json

```diff
@@ -4,7 +4,7 @@
   "protocol": "PROTOCOL_CONNECT",
   "codec": "CODEC_JSON",
   "host": "127.0.0.1",
-  "port": 51356,
+  "port": 53289,
```

Comment:
You might want to add a `--port` arg to server, that defaults to zero (to pick ephemeral port). That way you can start it up with a stable port, so these files are easier to work with.

Reply:
Yeah good call.  Added.